app/javascript/vue/protocol/step_attachments/mixins/wopi_file_modal.js

```diff
@@ -16,6 +16,7 @@ export default {
             $wopiModal.modal('hide');
             window.open(data.edit_url, '_blank');
             window.focus();
+            window.location.reload();
```

Comment:
Again why this?

Reply:
in a way it removes the need of the "reload file" need, but it also negates the whole ideology of not having the need of refreshing in vue.js.
will remove it